frontend/lab/samples/intro/move.evy

```diff
@@ -0,0 +1,17 @@
+color "purple"
+
+// Wrap a loop around the circle to
+// print 6 circles at x coordinates
+// 0 20 40 60 80 100:
+move 0 50
+circle 10
+
+// 🌶️🌶️
+// Can you change the program to make
+// the circle move from left to right?
+
+// 🌶️🌶🌶
```

Comment:
were the last two meant to be different?

Reply:
nope, fixed, thanks.